src/locators/jqueryLocator.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Provide jasmine locator
+ * @constructor
+ * @param {Config} config
+ * @param {Object} instanceConfig
+ * @param {Logger} logger
+ */
+function JQueryLocator(config, instanceConfig, logger) {
+  this.logger = logger;
+}
+
+/**
+ * Register jasmine locator
+ * @param {By} by - jasmine By object on which to add the new locator
+ */
+JQueryLocator.prototype.register = function (by) {
+  this.logger.debug('Registering jquery locator');
+  // http://angular.github.io/protractor/#/api?view=ProtractorBy.prototype.addLocator
+
+  by.addLocator('jq', function (query, opt_parentElement) {
```

Comment:
what about reusing the pattern with registering the controlLocator and so supporting the .toString() ?

Reply:
there is a default toString impl